packages/perseus/src/widget-ai-utils/categorizer/categorizer-ai-utils.test.ts

```diff
@@ -53,17 +53,18 @@ describe("Categorizer AI utils", () => {
     });
 
     it("it returns JSON with the expected format and fields", () => {
+        const userInput: PerseusCategorizerUserInput = {
+            values: [1, 0, 0, 1],
+        };
+
         const renderProps: any = {
             items: ["Luke Skywalker", "Darth Vader", "Yoda", "Han Solo"],
             categories: ["Galactic Empire", "Rebel Alliance"],
             values: [1, 0, 1, 1],
+            userInput,
         };
 
-        const userInput: PerseusCategorizerUserInput = {
-            values: [1, 0, 0, 1],
-        };
-
-        const resultJSON = getPromptJSON(renderProps, userInput);
+        const resultJSON = getPromptJSON(renderProps);
```

Comment:
All widgets receive user input from props now, so I refactored these functions to only take props rather than props + user input.